### PR TITLE
New Platforms for puppet-agent 1.3.4

### PR DESCRIPTION
**Pull request by shermdog**
As part of the puppet-agent 1.3.4 release Ubuntu 15.10 and cisco-wrlinux-7 were added.  This PR includes commits for both of those platforms, as well as enables the latest version of the packaging repo to be used for shipping.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,6 +11,6 @@ def vanagon_location_for(place)
 end
 
 gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || 'git@github.com:puppetlabs/vanagon#master')
-gem 'packaging', '0.4.2', :github => 'puppetlabs/packaging', :tag => '0.4.2'
+gem 'packaging', '~> 0.4.3', :github => 'puppetlabs/packaging'
```
Comment (underscorgan) [inline Gemfile]: You probably want ```~> 0.4``` here.

Comment (shermdog) [inline Gemfile]: Yeah... I think I do too.

File: configs/platforms/cisco-wrlinux-7-x86_64.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+platform "cisco-wrlinux-7-x86_64" do |plat|
+  plat.servicedir "/etc/init.d"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "sysv"
+  # The development environment provided by Cisco includes the build dependencies
+  # such as autoconf automake createrepo rsync gcc make rpm-build rpm-libs yum-utils
+  plat.yum_repo "http://pl-build-tools.delivery.puppetlabs.net/yum/cisco-wrlinux/7/pl-build-tools-cisco-wrlinux-7.repo"
+  plat.install_build_dependencies_with "yum install -y"
+  plat.vmpooler_template "cisco-wrlinux-7-x86_64"
+end
```
Comment (ScottGarman) [inline configs/platforms/cisco-wrlinux-7-x86_64.rb]: Need a newline at the end of the file?

Comment (shermdog) [inline configs/platforms/cisco-wrlinux-7-x86_64.rb]: Appropriate newlines are now there.  github may not be showing them properly, but they are now in line with the other platform defs.

**Comment by underscorgan** [comment]
:+1:

**Pull request merged**